e2e/bundle-size/scenarios/solid-start-full/src/routes/__root.tsx

```
@@ -0,0 +1,228 @@
+  createMiddleware,
+  createServerFn,
+  useServerFn,
+} from '@tanstack/solid-start'
+
+type BundleSizeKeep = {
+  hooksAndComponents: ReadonlyArray<unknown>
+  startSurface: ReadonlyArray<unknown>
+}
+
+declare global {
+  var __TANSTACK_BUNDLE_SIZE_KEEP__: BundleSizeKeep | undefined
```

ESLint

[error] Unexpected var, use let or const instead.

(no-var)